.github/workflows/ci.yml

```diff
@@ -10,19 +10,16 @@ jobs:
         node-version:
           - lts/-1
           - lts/*
-          - latest
+          #- latest
```

Comment:
node 22 have issues?

Reply:
Dang, no comment from me, I can't remember what the issue was, but the build looks like it had troubles: https://github.com/readmeio/markdown/actions/runs/9981242343/job/27584686162

Maybe it will just work now?

Reply:
Yup, cool.